client-rpc/src/lib.rs

```diff
@@ -1,8 +1,21 @@
-mod program;
-/// Used by c api
+use std::fmt::Debug;
+
 pub mod rpc;
-mod server;
 
-pub fn run() {
-    crate::program::run_electron();
+pub fn to_rpc_error<E: ToString + Debug>(error: E) -> jsonrpc_core::Error {
```

Comment:
`Display` is more idiomatic?
```suggestion
pub fn to_rpc_error<E: Display + Debug>(error: E) -> jsonrpc_core::Error {
```

Reply:
Maybe use `Error` directly here is more proper?

Reply:
Yeah. But, some error types may not implement `Error`. You can try that. If it compiles fine, then it’s good.

Reply:
Then I think `ToString + Debug` is actually accurate here, `ToString` for the `to_string` method, `Debug` for the logging.